lightning/src/ln/peer_handler.rs

```diff
@@ -1611,6 +1599,20 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 	}
 }
 
+fn is_gossip_msg(type_id: u16) -> bool {
+		match type_id {
+			msgs::ChannelAnnouncement::TYPE |
+			msgs::ChannelUpdate::TYPE |
+			msgs::NodeAnnouncement::TYPE |
+			msgs::QueryShortChannelIds::TYPE |
+			msgs::ReplyShortChannelIdsEnd::TYPE |
+			msgs::QueryChannelRange::TYPE |
+			msgs::ReplyChannelRange::TYPE |
+			msgs::GossipTimestampFilter::TYPE => true,
```

Comment:
IIUC, we don't want these to be considered "gossip" for logging purposes. @TheBlueMatt Could you confirm?

Reply:
Yea, I think we don't, probably better to leave them as TRACE for now and we can reduce them to GOSSIP later if its still too verbose. Probably only `Channel/NodeUpdate/Announcement`

Reply:
Okay, fixed with https://github.com/rust-bitcoin/rust-lightning/commit/7a2b716d26ae71b0020ab95b2ea3604f7f26c824.